pkg/cmd/operator/cleanupjob.go

```diff
@@ -118,11 +122,42 @@ func (o *CleanupJobOptions) Run(streams genericclioptions.IOStreams, cmd *cobra.
 	stopCh := signals.StopChannel()
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
+
 	go func() {
 		<-stopCh
 		cancel()
 	}()
 
+	var errs []error
+
+	err := o.runCleanup(ctx)
+	if err != nil {
```

Comment:
let's log that the cleanup call has finished so the logs give you context and timestamps

Reply:
done